numismatic/cli.py

```diff
@@ -262,39 +262,43 @@ def subscribe(state, feed, exchange, assets, currencies, interval, channels):
 
 
 @coin.command()
-@click.option('--market', '-m', default='all')
+@click.option('--subscriptions', '-s', default=['all'], type=str, multiple=True)
 @click.option('--event', 'stream', flag_value='event', default=True)
 @click.option('--raw', 'stream', flag_value='raw')
 @click.option('--collector', '-c', default='file', 
               type=click.Choice(Collector._get_subclasses().keys()))
 @click.option('--output', '-o', default='-', type=click.Path())
-@click.option('--filter', '-f', default='', type=str, multiple=True)
-@click.option('--type', '-t', default=None, multiple=True,
-              type=click.Choice(['None', 'Trade', 'Heartbeat', 'LimitOrder',
-                                 'CancelOrder']))
+@click.option('--filters', '-f', default=None, type=str, multiple=True)
+# FIXME: the types should be autogenerated from the Event types
```

Comment:
That would be cool... but it may also seem like magic...

Reply:
Perhaps, but we currently do the same for Feeds and Collectors. That's one of the nice things because you just have to register a new Feed and the CLI help output updates automatically.

Reply:
I know, that also seems like magic. The things I have against magic is that takes more time to understand how the infrastructure works as opposed to what it does. This is not to say it's always bad, but often times, it leads to over engineering...

Reply:
That's true.